crates/neon-runtime/src/napi/bindings/functions.rs

```diff
@@ -202,6 +202,17 @@ mod napi1 {
     );
 }
 
+#[cfg(feature = "napi-3")]
+mod napi3 {
+    use super::super::types::*;
+
+    generate!(
+        extern "C" {
+            fn fatal_exception(env: Env, err: Value) -> Status;
```

Comment:
This is a _fantastic_ API for adding panic catches. It allows you to trigger `uncaughtException` even when you aren't on a JavaScript stack.